src/components/application_manager/include/application_manager/application_manager_impl.h

```diff
@@ -1136,7 +1136,28 @@ class ApplicationManagerImpl
    */
   protocol_handler::MajorProtocolVersion SupportedSDLVersion() const OVERRIDE;
 
+  /**
+   * @brief Applies functor for each plugin
+   * @param functor Functor that will be applied to each plugin
+   */
+  void ForEachPlugin(
```

Comment:
@IGapchuk May be it is worth to replace function name with more detailed name something like ApplyFunctorForEachPlugin?

Reply:
@AByzhynar I agree with you. The name has been changed in 5a5d11ccdf1f25ae506f64a45f4fb13ab18bfcd1